### PR TITLE
Add more reference/credits to the project/EU programme that founded t…

### PR DESCRIPTION
**Pull request by zarch**
Add credits to:
- the [GRETA](http://www.alpine-space.eu/projects/greta/en/home) project,
- the European Regional Development  Fund that funded the project

Add scientific reference to the [G.POT](https://doi.org/10.1016/j.energy.2016.03.091) methodology implemented in the module to assess the GSHP potential.

### CODE REVIEW

File: grass7/raster/r.green/r.green.gshp/r.green.gshp.html [hunk 1]
```diff
@@ -5,12 +5,23 @@ <h2>DESCRIPTION</h2>
 planning, technical, economic/financial constraints).
 <p>
 
+The development of this tool has been developed within the  activities of the Alpine Space <a href="http://www.alpine-space.eu/projects/greta/en/home">GRETA</a> project.
```
Comment (neteler) [inline grass7/raster/r.green/r.green.gshp/r.green.gshp.html]:
```suggestion
The development of this tool has been developed within the  activities of the 
Alpine Space <a href="http://www.alpine-space.eu/projects/greta/en/home">GRETA</a> project.
```

Comment (neteler) [reply grass7/raster/r.green/r.green.gshp/r.green.gshp.html]: ... just breaking long lines as per HTML style guide.

Please merge then.

Comment (landam) [reply grass7/raster/r.green/r.green.gshp/r.green.gshp.html]: @zarch Please merge

Comment (veroandreo) [inline grass7/raster/r.green/r.green.gshp/r.green.gshp.html]:
```suggestion
The development of this tool was funded by the Alpine Space <a href="http://www.alpine-space.eu/projects/greta/en/home">GRETA</a> project.
```

Comment (veroandreo) [reply grass7/raster/r.green/r.green.gshp/r.green.gshp.html]: a suggestion to avoid repeating "development" in the same sentence

Comment (zarch) [reply grass7/raster/r.green/r.green.gshp/r.green.gshp.html]: Thank you all for the review, I'm going to merge this.

**Pull request merged**